backend/grants/models.py

```diff
@@ -37,6 +37,12 @@ class Status(models.TextChoices):
         Status.waiting_list_maybe.value,
     ]
 
+    BUDGET_STATUSES = [
```

Comment:
what does `budget` mean here?

Reply:
I moved it in the grant summary, as it's just for the budget calculations. 
We want to count only the "approved" / confirmed or those grant in `waiting_for_confirmtion`  in the allocated budget.